website/docs/guides/migration/from-v2-to-v3.md

```diff
@@ -0,0 +1,307 @@
+    - [1.2 Improved Mechanism for Signalling Message Processing Completion](#12-improved-mechanism-for-signalling-message-processing-completion)
+    - [1.3 Expose Worker Events to Client Applications](#13-expose-worker-events-to-client-applications)
+    - [1.4 Improve Dependency Injection Scope Management](#14-improve-dependency-injection-scope-management)
+  - [2. Improved Worker Distribution Strategy](#2-improved-worker-distribution-strategy)
+- [Conclusion](#conclusion)
+
+
+## Prerequisites
+
+As .NET Core 3.1 has reached the end of support in its lifecycle,  we have updated references to Core Packages (`Microsoft.*` and `System.*`) to version 6.
+
+While the KafkaFlow core and most of the extension packages are still targeting `netstandard2.0` which supports a range of runtimes, we recommend with this v3 update to target at least .NET 6 in applications using KafkaFlow.
+
+## Update package references
+
+To update to KafkaFlow v3, change the `Version` related to KafkaFlow packages to the latest v3 available in each project referencing KafkaFlow packages. 
+
+```
+<ItemGroup>
+-   <PackageReference Include="KafkaFlow" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.Abstractions" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.Admin" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.Admin.Dashboard" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.Admin.WebApi" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.BatchConsume" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.Compressor" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.Compressor.Gzip" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.Extensions.Hosting" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.LogHandler.Console" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.LogHandler.Microsoft" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.Microsoft.DependencyInjection" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.OpenTelemetry" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.SchemaRegistry" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.Serializer" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.Serializer.JsonCore" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.Serializer.NewtonsoftJson" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.Serializer.ProtobufNet" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.Serializer.SchemaRegistry.ConfluentAvro" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.Serializer.SchemaRegistry.ConfluentJson" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.Serializer.SchemaRegistry.ConfluentProtobuf" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.TypedHandler" Version="2.5.0" />
+-   <PackageReference Include="KafkaFlow.Unity" Version="2.5.0" />
+
```

Comment:
Shouldn't it be 2 different snippets?

Reply:
The [example](https://learn.microsoft.com/en-us/aspnet/core/migration/70-80?view=aspnetcore-8.0&tabs=visual-studio-code#update-package-references) @gsferreira gave in this [comment](https://github.com/Farfetch/kafkaflow/pull/471#discussion_r1400420530) is displaying as a Git Diff showing the lines removed and lines added.

But unlike this link we don't have any way to change the text color to better highlight that. 

But although we don't have text color, I think the "+" and "-" are explicit on the intent here.